stubs/arrayFunctions.stub

```diff
@@ -58,18 +58,31 @@ function uksort(array &$array, callable $callback): bool
 }
 
 /**
- * @template T of mixed
- *
+ * @template T
+ * @template U
  * @param array<T> $one
- * @param array<T> $two
- * @param callable(T, T): int $three
+ * @param array<U> $two
+ * @param callable(T, U): int $three
  */
 function array_udiff(
     array $one,
     array $two,
     callable $three
 ): int {}
 
+/**
+ * @template T
+ * @template U
+ * @param array<T> $one
+ * @param array<U> $two
+ * @param callable(T, U): int $three
+ */
+function array_uintersect(
+    array $one,
+    array $two,
+    callable $three
+): int {}
```

Comment:
Both `array_udiff` and `array_uintersect` return an array, not int: https://3v4l.org/R0WNi